app/assets/stylesheets/modules/links.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+a:link, a:visited, a:active {
+  text-decoration: none;
+  color: $white;
+  text-decoration:none;
+}
+
+a:hover {
+  color: $red;
+}
+
+form {
+  text-align: left;
+  color: $white;
+}
+
+a {
+  display: inline-block;
+  overflow: hidden;
+
+  vertical-align: top;
+
+  -webkit-perspective: 600px;
+  -moz-perspective: 600px;
+  -ms-perspective: 600px;
+  perspective: 600px;
+
+  -webkit-perspective-origin: 50% 50%;
+  -moz-perspective-origin: 50% 50%;
+  -ms-perspective-origin: 50% 50%;
+  perspective-origin: 50% 50%;
+}
+
+a span {
+  display: block;
+  position: relative;
+  padding: 0 2px;
+  border-radius: 5px;
+
+  -webkit-transition: all 400ms ease;
+  -moz-transition: all 400ms ease;
+  -ms-transition: all 400ms ease;
+  transition: all 400ms ease;
+
+  -webkit-transform-origin: 50% 0%;
+  -moz-transform-origin: 50% 0%;
+  -ms-transform-origin: 50% 0%;
+  transform-origin: 50% 0%;
+
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  -ms-transform-style: preserve-3d;
+  transform-style: preserve-3d;
+}
+
+a:hover span {
+  background: #00929c;
```

Comment:
Color literals like `#00929c` should only be used in variable declarations; they should be referred to via variable everywhere else.